index.js

```diff
@@ -14,7 +14,7 @@ const commandExists = path =>
   commandExists_(path).catch(_ => undefined);
 
 const getPathToElm = async () => {
-  const commands = ['./node_modules/.bin/elm', 'elm'];
+  const commands = [path.resolve('./node_modules/.bin/elm'), 'elm'];
   const CMD_NOT_FOUND_ERR = 'Could not find `elm` executable. You can install it with `yarn add elm` or `npm install elm`';
```

Comment:
Should we `console.log` this on line 27 before it throws (so debugging is better in the future)?

Reply:
Do you mean the `CMD_NOT_FOUND_ERR` error message? Throwing it will log it as an error message to the console anyway so it shouldn't be a problem.